styleguide.config.js

```diff
@@ -77,15 +77,31 @@ module.exports = {
       name: 'Components',
       content: './docs/components.md',
       skipComponentsWithoutExample: true,
-      sectionDepth: 1,
+      sectionDepth: 2,
```

Comment:
why ?

Reply:
@salesfelipe it's for the "show 1 per page" part.